lib/AST/ASTPrinter.cpp

```diff
@@ -315,6 +315,7 @@ PrintOptions PrintOptions::printSwiftInterfaceFile(ModuleDecl *ModuleToPrint,
     DAK_StaticInitializeObjCMetadata,
     DAK_RestatedObjCConformance,
     DAK_NonSendable,
+    TAK_retroactive
```

Comment:
Wow, the capitalization difference between `DAK_`s and `TAK_`s is kinda disgusting, but that’s not your fault.😅

Should we really leave `@retroactive` out of module interfaces? The Steering Group discussed the possibility of eventually having privileged-but-separate modules, like cross-import overlays or modules in the same package group, where `@retroactive` might be optional and its presence would affect the ABI; if that happened, we would certainly want all retroactive conformances marked in module interfaces.

(If you’re merely worried about backwards compatibility with older compilers, we now have an infrastructure for detecting when a declaration uses a new feature and printing an alternate version of it into the module interface using `#if`.)